common/lib/xmodule/xmodule/css/capa/display.scss

```
@@ -852,6 +852,15 @@ div.problem {
 
   }
 
+  .inputtype.formulaequationinput {
+    > div {
+      input {
+        direction: ltr;
+        text-align: inherit;
```

Before giving a recommendation on `text-align` why you're overriding it anyway?


----

I'm not really sure that `inherit` is a good pattern, it's the default behaviour anyway. Could you please use one of the following options:

To make it respect the RTL for the page:
```suggestion
        @include text-align(left);
```

Hard-coded left (I think this is the correct one).
```suggestion
        text-align: left;
```

Removing `inherit` implies `inherit` which could be the same as `@include text-align(left);`
```suggestion
```

---

Thank you @OmarIthawi , the alternative that fits here is `@include text-align(left);` . I need to overwrite text alignment because fixed `direction: ltr`  will make it always `left`. And I need it to be aligned to the right when we are in an RTL page